src/astro/sql/operators/export_file.py

```diff
@@ -91,9 +91,7 @@ def export_file(
     :param task_id: task id, optional
     """
 
-    task_id = (
-        task_id if task_id is not None else get_task_id("export_file", output_file.path)
-    )
+    task_id = task_id if task_id is not None else get_unique_task_id("export_file")
```

Comment:
What happens when `task_id` provided by user is not unique?

Reply:
@sunank200  get_unique_task_id() appends a suffix and convert it to a unique task id

Reply:
If the user provides task_id then it is expected that they would provide unique task_id if they user won't provide task_id then it will generate unique task_id

Reply:
@sunank200 it will raise the following error:
![Screenshot 2022-08-16 at 5 01 09 PM](https://user-images.githubusercontent.com/13021213/184869426-b54f1226-5675-4732-bd8d-f71217c91f53.png)

But I'm not sure if we should update the task_id provided by users or not. 
@kaxil @tatiana @feluelle thoughts?

Reply:
I think it is okay like that.

Btw you can also make it a bit simpler:
```suggestion
    task_id = task_id or get_unique_task_id("export_file")
```

Reply:
If the user does not provider a unique id an exception should be raised to the user.

Reply:
I'm also of the same opinion that we should raise the exception.